library/systemd/src/lib/yast2/service_configuration.rb

```diff
@@ -0,0 +1,194 @@
+require "yast"
+
+Yast.import "SystemdService"
+
+module Yast2
+  # Class that holds configuration for single or multiple services. It allows
+  # to read current status from system. It allows to modify target state. And it
+  # allows to write that target state so system reflects it.
+  # Its common usage is with {Yast2::ServiceWidget} which allows to modify
+  # configuration and write it when user approve whole dialog.
+  class ServiceConfiguration
+    include Yast::Logger
+
+    # services managed by this configuration
+    attr_reader :services
+
+    # creates new configuration that holds state for given services
+    # @param services<Yast::SystemdServiceClass::Service> services to configure
```

Comment:
Is this the documentation format correct? I guess it should be `services [<Yast::SystemdServiceClass::Service>] ...`

Reply:
yes, my fault. I still trying more to precise resulting design and document intentions and then test and document it properly.